test/test_torch.py

```diff
@@ -5977,6 +5978,24 @@ def _np_compare(self, fn_name, vals, device, dtype):
         torch_result = torch_fn(t).cpu()
         self.assertEqual(np_result, torch_result)
 
+    def _np_compare_func(self, fns, vals, device, dtype):
+        assert TEST_NUMPY
+
+        torch_fn, np_fn = fns
+
+        a = np.array(vals, dtype=torch_to_numpy_dtype_dict[dtype])
+        np_result = torch.from_numpy(np_fn(a).copy())
```

Comment:
Why do you need to make a `.copy()` of the result?

Reply:
As flip manipulates the strides and `numpy` supports negative strides, 
without the `.copy()` we get, 
```
ValueError: At least one stride in the given numpy array is negative, and tensors with negative strides are not currently supported. (You can probably work around this by making a copy of your array  with array.copy()
```

Reply:
Would you add a comment explaining that?

Reply:
Done.